handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslEngine.java

```diff
@@ -2199,6 +2209,10 @@ public int getPeerPort() {
 
         @Override
         public int getPacketBufferSize() {
+            // TODO: Do a better job for TLSv1.3 here.
```

Comment:
@carl-mastrangelo @ejona86 @ryanoneill @bryce-anderson @Scottmitch @trustin this needs to be fixed before merging as it may waste memory.

Reply:
Currently checking what to do best...